migrate/20250318_add_maintenance_window.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:postgres_resource) do
+      add_column :maintenance_window_start_at, :integer
```

Comment:
I think it might be worthwhile to add a check constraint:

```ruby
constraint :valid_maintenance_windows_start_at, maintenance_window_start_at: 0..23
```